docs/standard/serialization/toc.yml

```diff
@@ -73,23 +75,12 @@
       href: add-element-for-schemaimporterextensions.md
     - name: <xmlSerializer> Element
       href: xmlserializer-element.md
-    - name: Serialization Tools
+    - name: Tools
       href: serialization-tools.md
       items:
       - name: XML Serializer Generator Tool (Sgen.exe)
         href: xml-serializer-generator-tool-sgen-exe.md
       - name: XML Schema Definition Tool (Xsd.exe)
         href: xml-schema-definition-tool-xsd-exe.md
-    - name: Serialization Samples
-      href: serialization-samples.md
-      items:
-      - name: Basic Serialization Technology Sample
-        href: basic-serialization-technology-sample.md
-      - name: Custom Serialization Order With XmlSerializer
-        href: custom-serialization-order-with-xmlserializer.md
-      - name: SchemaImporterExtension Technology Sample
-        href: schemaimporterextension-technology-sample.md
-      - name: Version Tolerant Serialization Technology Sample
-        href: version-tolerant-serialization-technology-sample.md
-      - name: Web Services Generics Serialization Technology Sample
-        href: web-services-generics-serialization-technology-sample.md
+    - name: Samples
+      href: serialization-samples-xml.md
```

Comment:
why remove the children articles from the TOC here?

Reply:
One of the samples is for both XML and binary.  If I kept them in the TOC I would have had to duplicate that node.